source/Halibut/Transport/Protocol/MessageExchangeProtocol.cs

```diff
@@ -183,8 +183,8 @@ public async Task ExchangeAsServerSynchronouslyAsync(Func<RequestMessage, Respon
 
         public async Task ExchangeAsServerAsync(Func<RequestMessage, Task<ResponseMessage>> incomingRequestProcessor, Func<RemoteIdentity, IPendingRequestQueue> pendingRequests, CancellationToken cancellationToken)
```

Comment:
Note that I didn't add the logging to the sync paths. Should I?

Reply:
We are really weighing up leaving the existing code alone, vs being able to log with and without the feature toggle on.

🤔 Is this extra info worth the risk of touching existing code.

@nathanwoctopusdeploy, do you have a preference?

Reply:
We are only really interested in async errors. The only thing we would lose is the ability to see if this issue was already happening in sync

Reply:
Happy either way though

Reply:
Let's go with the safe approach then, and leave the sync path alone.